pyproject.toml

```diff
@@ -48,6 +48,7 @@ beautifulsoup4 = "^4.12.3"
 pandas = "^2.1.4"
 marko = "^2.1.2"
 openpyxl = "^3.1.5"
+lxml = ">=4.8.0"
```

Comment:
Only remark here is, please protect against moving to next major, by using `^4.8.0`